docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -21,60 +42,82 @@ For secure communication between APM Server and APM Agents, see <<apm-secure-age
 
-* <<apm-privileges-to-publish-events,Writer role>>: To publish events collected by APM Server.
-* <<apm-privileges-to-publish-monitoring,Monitoring role>>: One for sending monitoring
-information, and another for viewing it.
-* <<apm-privileges-api-key,API key role>>: To create and manage API keys.
-* <<apm-privileges-agent-central-config,Central configuration management role>>: To view
-APM Agent central configurations.
-* <<apm-privileges-rum-source-mapping,RUM source mapping role>>: To read RUM source maps.
+[TIP]
+====
+A _role_ identifies a set of permissions that translate to privileges on resources. You can associate a _user or group_ with an arbitrary number of roles. The total set of permissions that a user has is defined by the union of the permissions in all its roles.
 
-{es-security-features} provides {ref}/built-in-roles.html[built-in roles] that grant a
-subset of the privileges needed by APM users.
-When possible, assign users the built-in roles to minimize the affect of future changes on your security strategy.
-If no built-in role is available, you can assign users the privileges needed to accomplish a specific task.
-In general, there are three types of privileges you'll work with:
+In general, there are three types of privileges you'll work with when creating roles:
 
 * **{es} cluster privileges**: Manage the actions a user can perform against your cluster.
 * **{es} index privileges**: Control access to the data in specific indices your cluster.
 * **{kib} space privileges**: Grant users write or read access to features and apps within {kib}.
+====
+
+Below are some common roles that APM Server users might need:
+
+* <<apm-privileges-to-publish-events,*Writer role*>>:
+Allows a user to publish events collected by APM Server, which is *required* to write to {es}.
+* <<apm-privileges-agent-central-config,*Central configuration management role*>>:
+Allows a user to view APM Agent central configurations, which is *required* when
+<<apm-agent-configuration,central configuration management>> is enabled (it is enabled by default).
+* <<apm-privileges-to-publish-monitoring,*Monitoring role*>>: Allows a user to publish monitoring data,
+view monitoring data, or both.
+* <<apm-privileges-rum-source-mapping,*RUM source mapping role*>>: Allows a user to read RUM source maps.
+
+// TO DO: Replace <do something> with a common task made possible by the listed roles
+.Example: Assigning multiple roles to an APM Server user
+*****
+If you use agent central configuration to manage APM agents, you use RUM source mapping,
+and you want to create an APM Server user who can <do something>,
+you would need to assign these three roles to the user:
+
+* <<apm-privileges-to-publish-events,Writer role>>
+* <<apm-privileges-agent-central-config,Central configuration management role>>
+* <<apm-privileges-to-publish-events,RUM source mapping role>>
+*****
```

Comment:
What is an example of a task that an APM Server user with these roles might need to complete? Feel free to suggest a different set of roles as an example if this one doesn't make sense.

Reply:
How does this look?

```suggestion
.Example: Assigning multiple roles to an APM Server user
*****
If you want to create an APM Server user who can use the Elastic APM Real User Monitoring (RUM)
JavaScript Agent to ingest data from a frontend application and you use central configuration
to manage APM agents, you would need to assign these three roles to the user:

* <<apm-privileges-to-publish-events,Writer role>>
* <<apm-privileges-agent-central-config,Central configuration management role>>
* <<apm-privileges-to-publish-events,RUM source mapping role>>
*****
```

Reply:
Looks good :+1: ! Just one small detail, on line 94 I see `<<apm-privileges-to-publish-events,RUM source mapping role>>`, is `apm-privileges-to-publish-events` the correct link path?

Reply:
Good catch!